aarch64_extension.cpp

```diff
@@ -275,6 +328,108 @@ class AArch64ArchitectureExtension : public ArchitectureHook {
     return false;
   }
 
+  bool LiftFMOV(cs_insn* instr, LowLevelILFunction& il) {
```

Comment:
This instruction has 3 variants, [FMOV (register)](https://developer.arm.com/docs/100069/0607/a64-floating-point-instructions/fmov-register),  [FMOV (general)](https://developer.arm.com/docs/100069/0607/a64-floating-point-instructions/fmov-general) and [FMOV (scalar, immediate)](https://developer.arm.com/docs/100069/0607/a64-floating-point-instructions/fmov-scalar-immediate). We should definitely be handling these cases (or at a minimum just the register and general case).